### DISCUSSION TITLE
Feature request: add pushover to notification channel

### DISCUSSION THREAD

**Discussion by nalakawula** [hyperjumptech/monika#658] [Ideas]
I would like to request to add [pushover](https://pushover.net/) as notification channel. My team already use pushover for years, their service is good in our opinion.

Thanks

**Comment by nicnocquee** [comment]
First time I heard about it. Looks interesting! I made an [issue](https://github.com/hyperjumptech/monika/issues/661).

**Reply by nicnocquee** [reply]
Hi @nalakawula we have released a new version that includes the support for pushover. Let us know if something's not working.

**Reply by nalakawula** [reply]
Thank you @nicnocquee , I'll try it.

**Reply by nalakawula** [reply]
Work as expected, Thank you!
